Tools/Create-Azure-Sentinel-Solution/input/Solution_CohesitySecurity.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Name": "CohesitySecurity",
+    "Author": "Microsoft - support@microsoft.com",
+    "Description": "TBD",
+    "Data Connectors": [
+        "Data Connectors/helios_connector.json"
+    ],
+    "Analytic Rules": [
+        "Analytic Rules/RansomwareAlert.yaml"
+    ],
+    "Playbooks": [
+        "Playbooks/Incident_Email_Playbook/azuredeploy.json",
+        "Playbooks/Incident_VM_Playbook/azuredeploy.json",
+        "Playbooks/SNOW-CreateAndUpdateIncident/azuredeploy.json"
+    ],
+    "BasePath": "Solutions/CohesitySecurity",
+    "Version": "2.0.1",
```

Comment:
Please can you change the version to 2.0.0 as this the first time, the solution would be published, thanks.

Reply:
The file was removed